<launch>
	<node name="dsl_grid3d" pkg="dsl_gridsearch" type="dsl_grid3d_node" output="screen">
		<param name="mesh_filename" type="string" value="$(find dsl_gridsearch)/data/hackerman2.stl"/>
		<param name="use_textured_mesh" type="bool" value="true"/>
		<param name="cells_per_meter" type="double" value="1"/>
		<!--<param name="grid_length" type="int" value="500"/>
		<param name="grid_width" type="int" value="500"/>
		<param name="grid_height" type="int" value="10"/>
		<param name="grid_zmin" type="double" value="-8"/>
		<param name="grid_ymin" type="double" value="-150"/>
		<param name="grid_xmin" type="double" value="-150"/>-->
	</node>
</launch>
